lazy_tensor_core/lazy_tensor_core/csrc/tensor_aten_ops.cpp

```diff
@@ -322,7 +322,10 @@ torch::lazy::LazyTensor squeeze(const torch::lazy::LazyTensor& input) {
   auto input_shape = input.shape();
   auto output_dimensions = ir::ops::BuildSqueezedDimensions(
       input_shape.Get().sizes(), /*squeeze_dim=*/-1);
-  return view(input, output_dimensions);
```

Comment:
any reason we don't just call the below squeeze(input, -1) instead of duplicating?